Dapper.Tests.Performance/Dapper.Tests.Performance.csproj

```diff
@@ -17,6 +17,8 @@
     <!--<PackageReference Include="BLToolkit" Version="4.3.6" />-->
     <PackageReference Include="EntityFramework" Version="6.1.3" />
     <PackageReference Include="FirebirdSql.Data.FirebirdClient" Version="5.9.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="1.1.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="1.1.2" />

```

Comment:
Since `Microsoft.EntityFrameworkCore.SqlServer` ultimately depends on `Microsoft.EntityFrameworkCore` (via `.Relational`), this line isn't needed in the new project system:
```xml
<PackageReference Include="Microsoft.EntityFrameworkCore" Version="1.1.2" />
```